### PR TITLE
Swift 4.2 support

### PR DESCRIPTION
**Pull request by djbe**
The codebase should still be compatible with Swift 4.1.

### CODE REVIEW

**Comment by Cee** [comment]
Hang tight! I'll check this tonight :)

**Comment by Cee** [comment]
LGTM.

File: TagListView/TagView.swift [hunk 1]
```diff
@@ -216,3 +216,13 @@ open class TagView: UIButton {
         }
     }
 }
+
+/// Swift < 4.2 support
```
Comment (Cee) [inline TagListView/TagView.swift]: Is it better to bring these extensions to the beginning of this file?

File: TagListView/TagView.swift [hunk 2]
```diff
@@ -216,3 +216,13 @@ open class TagView: UIButton {
...
+#if !(swift(>=4.2))
+private extension NSAttributedString {
+    typealias Key = NSAttributedStringKey
```
Comment (Cee) [inline TagListView/TagView.swift]: Shall we use lowercased `key` or just keep this one?

Comment (djbe) [reply TagListView/TagView.swift]: This is for compatibility with Swift 4.2, where it is uppercase `Key`.

File: TagListView/TagView.swift (hunk 1 above)
Comment (djbe) [reply TagListView/TagView.swift]: Can do that if you'd prefer.

**Comment by Cee** [comment]
GJ! 

cc @xhacker Let's make an update for CocoaPods.

**Pull request merged**